include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -104,65 +104,77 @@ cigar_op compare_aligned_values(reference_char_type const reference_char,
                         : 'M'_cigar_op;
 }
 
-/*!\brief Transforms an alignment represented by two aligned sequences into the
- *        corresponding CIGAR string.
+/*!\brief Creates a CIGAR string (SAM format) given an alignment represented by two aligned sequences.
  * \ingroup alignment_file
  *
- * \tparam ref_seq_type    Must model std::ranges::forward_range. The value_type must
- *                         be equality comparable to seqan3::gap.
- * \tparam query_seq_type  Must model std::ranges::forward_range. The value_type must
- *                         be equality comparable to seqan3::gap.
- * \param  ref_seq         The reference sequence to compare against the query sequence.
- * \param  query_seq       The query sequence to build the CIGAR string for.
+ * \tparam alignment_type  Must model the seqan3::tuple_like and must
+ *                         have std::tuple_size 2. Each tuple element must model
+ *                         std::forward_range and its value_type must be comparable
+ *                         to seqan3::gap.
+ * \param  alignment       The alignment, represented by a pair of aligned sequences,
+ *                         to be transformed into CIGAR_vector based on the
+ *                         second (query) sequence.
  * \param  query_start_pos The start position of the alignment in the query
  *                         sequence indicating soft-clipping.
  * \param  query_end_pos   The end position of the alignment in the query
  *                         sequence indicating soft-clipping.
  * \param  extended_cigar  Whether to print the extended cigar alphabet or not. See CIGAR operation.
- * \returns An std::string representing the alignment as a CIGAR string.
+ * \returns An std::vector<seqan3::cigar> representing the alignment.
  *
- * ### Theoretical Example:
+ * \details
  *
- * The following alignment reference sequence on top and the query sequence at
- * the bottom.
+ * \attention Note that CIGAR elements (respectively by their
```

Comment:
Because I like it more compact and think 120 line length is too long.. and since documentation is :nail_care: I thought it is not a problem but personal flavour.

Reply:
In my opinion code style, counting in comments and API documentation, should follow a style guide and not personal preferences to avoid humble mumble in the code. I know that this was never documented appropriately but this seems to me like a relatively new flavor that was never done before. So I would recommend to stick to the common style and use the 120 line lengths. Before introducing new styles I would prefer discussing it. So to avoid confusions as we had when a subgroup of people decided putting complex nested classes out of its enclosing class body, which was not communicated properly 😅 .

Reply:
fine